src/Symfony/Component/HttpKernel/Event/GetResponseForExceptionEvent.php

```diff
@@ -44,6 +40,21 @@ public function __construct(HttpKernelInterface $kernel, Request $request, int $
      * @return \Exception The thrown exception
      */
     public function getException()
+    {
+        if (!$this->exception instanceof \Exception) {
+            return new ErrorException($this->exception);
+        }
+
+        return $this->exception;
+    }
+
+    /**
+     * @internal to be removed in Symfony 5.0
+     *
+     * Use this method when the next possible code paths are fully compatible
```

Comment:
it's internal and the description seems irrelevant. what I meant was something like `@internal to be removed in Symfony 5.0 when getException() can return a Throwable instead of an Exception

Reply:
The comment was targeted for Symfony contributors 😄

Reply:
It explains when to use this method instead of `getException()` on 4.4.